pages/windows/sfc.md

```diff
@@ -0,0 +1,31 @@
+# sfc
+
+> Scans the integrity of Windows system files.
+
+- Display information about the usage of the command:
+
+`sfc`
+
+- Scan all system files and, if possible, repair any problems:
+
+`sfc /scannow`
+
+- Scan all system files without attempting to repair any:
+
+`sfc /verifyonly`
+
+- Scan a specific file and, if possible, repair any problems:
+
+`sfc /scanfile={{path/to/file}}`
+
+- Scan a specific file without attempting to repair it:
+
+`sfc /verifyfile={{path/to/file}}`
+
+- When repairing offline, specify the boot directory:
```

Comment:
What does it mean by "repairing offline" ?

Reply:
This is where it is being run from either a boot disk, or from a remote computer. It allows you to point at the Windows paths if you are unable to boot into Windows.

I guess the other thing this may be useful in is for checking Windows as a sysadmin.

Reply:
ok